client-v2/src/shared/components/SnapLink.tsx

```diff
@@ -36,34 +38,39 @@ interface ContainerProps extends SnapLinkProps {
   selectSharedState?: (state: any) => State;
 }
 
-const SnapLink = ({ id, fade, onClick, size = 'default', onDelete, ...rest }: SnapLinkProps) => (
+const SnapLink = ({
+  id,
+  fade,
+  onClick,
+  isSnapLink,
```

Comment:
Is there a reason we have an isSnapLink prop here? (This is a snaplink component - surely we know we're a snap link! ...but I may be missing something 😄 )